public/scripts/clientContactServer.js

```diff
@@ -1,20 +1,14 @@
 
 const MESSAGE = document.querySelector('.message');
 
-async function sendData(data){
+async function sendData(data,method){
```

Comment:
love to see this. since you are adding the http method as a parameter, you've turned the `sendData` func into more of a utility, which is great. This is a very common pattern and good abstraction. 

instead of `sendData` , I'd suggest `sendRequest`